src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.ProductDao;
+import core.basesyntax.dao.ProductDaoImpl;
+import core.basesyntax.model.Product;
+import core.basesyntax.service.ReportService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    private static final ProductDao<Product, Integer> dao = new ProductDaoImpl();
+    private static final String TITLE = "fruit,quantity";
+
+    @Override
+    public List<String> createReport() {
+        List<String> res = new ArrayList<>(List.of(TITLE));
```

Comment:
please try to not cut the variables names:
```suggestion
        List<String> result = new ArrayList<>(List.of(TITLE));
```